public/index.html

```diff
@@ -48,7 +48,9 @@ <h2 class="panel__title">Visitors Table</h2>
             <th class="visitors__hcell">Longitude</th>
             <th class="visitors__hcell">Date</th>
           </thead>
-          <tbody class="visitors__body"></tbody>
+          <tbody class="visitors__body">
+
```

Comment:
?

Reply:
Didn't see this yesterday, sorry.. 🙈
There is no issue in tables..